src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java

```diff
@@ -45,7 +45,7 @@
     private static final List<String> staticJavaLibs = Arrays.asList(
             "j2pkcs11", "java", "net", "nio", "prefs", "sunec", "zip");
     private static final List<String> staticJvmLibs = Arrays.asList(
-            "ffi", "jvm", "libchelper", "strictmath");
+            "ffi", "jvm", "fdlibm", "libchelper");
```

Comment:
The `fdlibm` library should probably be moved to the `staticJavaLibs` list to be consistent with other targets.

Reply:
moved